src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md

```diff
@@ -48,8 +48,10 @@ Set-AzVMDiskEncryptionExtension [-ResourceGroupName] <String> [-VMName] <String>
 
 ## DESCRIPTION
 The **Set-AzVMDiskEncryptionExtension** cmdlet enables encryption on a running infrastructure as a service (IaaS) virtual machine in Azure.
-This cmdlet enables encryption by installing the disk encryption extension on the virtual machine.
-If no *Name* parameter is specified, an extension with the default name AzureDiskEncryption for virtual machines that run the Windows operating system or AzureDiskEncryptionForLinux for Linux virtual machines are installed.
+This cmdlet enables encryption by installing the disk encryption extension on the virtual machine. 
+
+The *VolumeType* parameter is required when encrypting Linux virtual machines, but can be omitted when encrypting Windows virtual machines.
```

Comment:
VolumeType is required for Linux, but it also must match what the distribution supports.   For example if a Linux distribution only supports data disk encryption, the Volume type specified must be Data.